tests/real/community.rs

```diff
@@ -62,6 +62,77 @@ fn roundtrip() {
     assert_eq!(job.args(), &[Value::from("z")]);
 }
 
+#[test]
+#[cfg(feature = "tls")]
+fn roundtrip_tls() {
+    use native_tls::{Certificate, TlsConnector};
+    use std::{env, fs::File, io::Read};
+
+    if env::var_os("FAKTORY_URL").is_none() || env::var_os("FAKTORY_URL_SECURE").is_none() {
```

Comment:
I think this should only return early if `FAKTORY_URL_SECURE` is unset, right? If `FAKTORY_URL` is unset, this test can still run.

Reply:
Indeed, fixed